wai-app-static/WaiAppStatic/Storage/Filesystem.hs

```diff
@@ -120,8 +119,8 @@ webAppLookup hashFunc prefix pieces =
 -- exists.
 hashFile :: FilePath -> IO ByteString
 hashFile fp = do
-    h <- Crypto.Hash.Conduit.hashFile fp
-    return $ B64.encode $ toBytes (h :: Digest MD5)
+    f <- BL.readFile fp
```

Comment:
The correct way to do this is something like:

```haskell
withBinaryFile fp ReadMode $ \h -> do
    lbs <- L.hGetContents h
    ...
```

The problem with `readFile` rears its head in the case of async exceptions. It's an uncommon case to be sure, but I try to avoid the lazy I/O functions in libraries anyway to prevent confusing behavior.